extensions/monday/src/lib/api.tsx

```diff
@@ -86,6 +86,45 @@ export async function getUser(): Promise<Me> {
   return (result as Record<string, unknown>).me as Me;
 }
 
+export async function getBoardAndUser(): Promise<BoardsResponse> {
```

Comment:
```suggestion
export async function getQuickAddBoardAndUser(): Promise<BoardsResponse> {
```

or, pass the id along to make it a tad more reusable:

```suggestion
export async function getBoardAndUser(boardId: number): Promise<BoardsResponse> {
```